tests/unit/Core/Framework/Demodata/Faker/CommerceTest.php

```diff
@@ -0,0 +1,28 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Tests\Unit\Core\Framework\Demodata\Faker;
+
+use Faker\Factory;
+use PHPUnit\Framework\Attributes\CoversClass;
+use PHPUnit\Framework\TestCase;
+use Shopware\Core\Framework\Demodata\Faker\Commerce;
+
+/**
+ * @internal
+ */
+#[CoversClass(Commerce::class)]
+class CommerceTest extends TestCase
+{
+    public function testCustomFieldSet(): void
+    {
+        $commerce = new Commerce(Factory::create());
+
+        $property = new \ReflectionProperty($commerce, 'productName');
```

Comment:
you may want to use `\Shopware\Core\Framework\Test\TestCaseHelper\ReflectionHelper::getProperty` instead